pkg/jx/cmd/create/create_devpod.go

```diff
@@ -814,7 +815,7 @@ func (o *CreateDevPodOptions) Run() error {
 	}
 
 	if o.Sync {
-		syncOptions := &SyncOptions{
+		syncOptions := &sync.SyncOptions{
```

Comment:
In another round of refactoring we can remove the command name from the option:

`sync.SyncOptions` => `sync.Options`